### PR TITLE
Added Simple API workflow structure

### PR DESCRIPTION
**Pull request by stephenwf**
@elrossco22

### CODE REVIEW

**Comment by stephenwf** [comment]
@thewilkybarkid This request the fix we were talking about this morning with the JsonEncoder, do you want me to raise a PR with the ApiSdk repo?

File: tests/src/Search/Workflow/GetSerializer.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+trait GetSerializer
+{
+    private $serializer;
+
+    public function getSerializer()
+    {
+        if (!method_exists($this, 'getHttpClient')) {
+            throw new LogicException('You need to include the HttpClient to initialize the serializer');
+        }
```
Comment (thewilkybarkid) [inline tests/src/Search/Workflow/GetSerializer.php]: This trait should `use` the other one.

**Comment by thewilkybarkid** [comment]
@stephenwf Please do

File: tests/src/Search/Workflow/GetSerializer.php (hunk 1 above)
Comment (stephenwf) [inline tests/src/Search/Workflow/GetSerializer.php]: I had that initially but then when I was creating the Mock trait here: 
https://github.com/elifesciences/search/pull/13/files/20e8dc5dd227b86889fc3e2c968b6c0ad3146efe#diff-92d16925d99a8a21d6f7bb0638029052R42

I realised I had a double-dependency, what would you suggest?

File: src/Search/Api/Elasticsearch/ElasticSearchResponse.php [hunk 2]
```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace eLife\Search\Api\Elasticsearch;
+
+
+use eLife\Search\Api\Query\int;
```
Comment (elrossco22) [inline src/Search/Api/Elasticsearch/ElasticSearchResponse.php]: Can we remove this, if it isn't used please

**Pull request merged**